<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ramzi</class>
 <widget class="QMainWindow" name="Ramzi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="central" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1200</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QComboBox" name="comboBox_8">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>10</y>
       <width>301</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 85, 255) ;
background-color: rgb(49, 49, 49);</string>
     </property>
     <item>
      <property name="text">
       <string>Seance</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/bakground/patienticon.png</normaloff>:/bakground/patienticon.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cours</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/bakground/rendez_vousicon.png</normaloff>:/bakground/rendez_vousicon.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Statistique</string>
      </property>
     </item>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>67</y>
       <width>1191</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stackedWidgetPage1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>531</width>
         <height>421</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string>Ajouter Seance</string>
        </attribute>
        <widget class="QPushButton" name="SupprimerSc">
         <property name="geometry">
          <rect>
           <x>179</x>
           <y>360</y>
           <width>201</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
        <widget class="QLabel" name="reference_sup_6">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>330</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>matricule</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_63">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>300</y>
           <width>321</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Saisir l'id de seance a supprimer :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_9">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_10">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>210</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>140</y>
           <width>251</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>220</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Duree</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ajouterSC">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>250</y>
           <width>121</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_4">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>330</y>
           <width>201</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_9">
        <attribute name="title">
         <string>Modifier Seance</string>
        </attribute>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>60</y>
           <width>281</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>170</y>
           <width>281</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_11">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>270</y>
           <width>281</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ModifieSC">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>340</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>contact Etudiant </string>
        </attribute>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>371</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>80</y>
           <width>211</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>110</y>
           <width>371</width>
           <height>221</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>350</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>envoyer</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Mail</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Sujet</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Message</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QTableView" name="tabSeance">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>70</y>
         <width>611</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>40</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>recherhe par date</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>1090</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="stackedWidgetPage2">
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>581</width>
         <height>511</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_15">
        <attribute name="title">
         <string>ajouter Cours</string>
        </attribute>
        <widget class="QLabel" name="label_64">
         <property name="geometry">
          <rect>
           <x>31</x>
           <y>340</y>
           <width>321</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Saisir l'id du  cours a supprimer :</string>
         </property>
        </widget>
        <widget class="QPushButton" name="supprimerC_2">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>390</y>
           <width>201</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
        <widget class="QLabel" name="reference_sup_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>370</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>100</y>
           <width>221</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>160</y>
           <width>221</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>220</y>
           <width>221</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>100</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>170</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>230</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QPushButton" name="AjouterC">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>290</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>360</y>
           <width>201</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_16">
        <attribute name="title">
         <string>Modifier Cours</string>
        </attribute>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>311</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>180</y>
           <width>311</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>290</y>
           <width>311</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>300</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ModifierC">
         <property name="geometry">
          <rect>
           <x>304</x>
           <y>360</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QTableView" name="tabcours">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>60</y>
         <width>581</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>30</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>DESC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>1070</x>
         <y>22</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TRIE</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>811</x>
         <y>30</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prenom</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>681</width>
         <height>331</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
